src/main/java/com/bookappstore/model/CartItem.java

```diff
@@ -0,0 +1,34 @@
+package com.bookappstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.SQLRestriction;
+
+@Entity
+@Setter
+@Getter
+@SQLDelete(sql = "UPDATE cart_item SET is_deleted = true WHERE = id=?")
+@SQLRestriction("is_deleted = false")
+@Table(name = "cart_item")
```

Comment:
Tables should be named in plural 
```suggestion
@Table(name = "cart_items")
```